docs/guide-configs.md

```diff
@@ -21,39 +21,312 @@ Below you will find pre-built configuration files for the following TRaSH Guides
-[anime-sonarr-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/anime-sonarr-v4.yml
-[french-web-1080p-multi-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/french-web-1080p-multi-v4.yml
-[french-web-1080p-vostfr-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/french-web-1080p-vostfr-v4.yml
-[french-anime-multi-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/french-anime-multi-v4.yml
-[french-anime-vostfr-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/french-anime-vostfr-v4.yml
+| Radarr                                                             | Sonarr v4                                              |
+| ------------------------------------------------------------------ | ------------------------------------------------------ |
+| [HD Bluray + WEB](#hd-bluray--web)                                 | [WEB-1080p](#web-1080p-v4)                             |
+| [UHD Bluray + WEB](#uhd-bluray--web)                               | [WEB-2160p](#web-2160p-v4)                             |
+| [Remux + WEB 1080p](#remux--web-1080p)                             | [Anime (Sonarr)](#anime-sonarr-v4)                     |
+| [Remux + WEB 2160p](#remux--web-2160p)                             | [French WEB-1080p MULTi](#french-web-1080p-multi-v4)   |
+| [Anime (Radarr)](#anime-radarr)                                    | [French WEB-1080p VOSTFR](#french-web-1080p-vostfr-v4) |
+| [French HD Bluray + WEB MULTi](#french-hd-bluray--web-multi)       | [French Anime MULTi](#french-anime-multi-v4)           |
+| [French HD Bluray + WEB VOSTFR](#french-hd-bluray--web-vostfr)     | [French Anime VOSTFR](#french-anime-vostfr-v4)         |
+| [French UHD Bluray + WEB MULTi](#french-uhd-bluray--web-multi)     |                                                        |
+| [French UHD Bluray + WEB VOSTFR](#french-uhd-bluray--web-vostfr)   |                                                        |
+| [French Remux + WEB 1080p MULTi](#french-remux--web-1080p-multi)   |                                                        |
+| [French Remux + WEB 1080p VOSTFR](#french-remux--web-1080p-vostfr) |                                                        |
+| [French Remux + WEB 2160p MULTi](#french-remux--web-2160p-multi)   |                                                        |
+| [French Remux + WEB 2160p VOSTFR](#french-remux--web-2160p-vostfr) |                                                        |
+
+---
+
+## Radarr
+
+### HD Bluray + WEB
+
+<details>
+<summary>Click to show/hide</summary>
+<div>
+
+```yml reference title="Copy this configuration into your own configuration file:"
+https://github.com/recyclarr/config-templates/blob/master/radarr/hd-bluray-web.yml
+```
+
+</div>
+</details>
+
+### UHD Bluray + WEB
+
+<details>
+<summary>Click to show/hide</summary>
+<div>
```

Comment:
Question: What does the `<div>` do here? Does it change the appearance/output somehow?

Reply:
> Question: What does the `<div>` do here? Does it change the appearance/output somehow?

As it turns out, nothing 😄 have removed all of them with another commit.